mcsema/BC/Lift.h

```diff
@@ -43,6 +43,7 @@ struct TranslationContext {
   const NativeInstruction *cfg_inst;
   llvm::Function *lifted_func;
   std::unordered_map<uint64_t, llvm::BasicBlock *> ea_to_block;
+  std::unordered_map<uint64_t, llvm::BasicBlock *> lp_to_block;
```

Comment:
Is this mapping which block falls to which landingpad?

Reply:
It maps the `landingpad` ea with the block.